recipes/arrow-c-glib/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "arrow-c-glib" %}
+{% set version = "8.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://dist.apache.org/repos/dist/release/arrow/arrow-{{ version }}/apache-arrow-{{ version }}.tar.gz
+  sha256: ad9a05705117c989c116bae9ac70492fe015050e1b80fb0e38fde4b5d863aaa3
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - gnuconfig
+    - ninja
+  host:
+    - meson
+    - arrow-cpp
+    - glib
+
+about:
+  home: https://arrow.apache.org
+  summary: 'Arrow GLib is a wrapper library for Arrow C++. Arrow GLib provides C API.'
+  description: |
+    Apache Arrow is a development platform for in-memory analytics.
+    It contains a set of technologies that enable big data systems
+    to process and move data fast. Arrow GLib is a wrapper library for Arrow C++.
+    Arrow GLib provides C API.
+  license: Apache-2.0
+  license_family: Apache
+  license_file: LICENSE.txt
+  doc_url: https://arrow.apache.org/docs/c_glib/
+  dev_url: https://github.com/apache/arrow/tree/master/c_glib
+
+extra:
+  recipe-maintainers:
+    - xmnlab
```

Comment:
Can I also be a maintainer here? I take care of all the other Arrow stuff so I would be happy to help

Reply:
Sure thing. I will do it in a bit :) thanks